### PR TITLE
Refactored (renamed) Initial views, view models, and controllers to make them TeamUseCase 

### PR DESCRIPTION
**Pull request by soniavaidya05**
I refactored the Initial Views and Initial interface adapters into a Team Use Case instead. The initial view was supposed to be login and signup, but we ended up implementing the Team use case GUI first. Renamed it for clarity when I add more views for more use cases.

### CODE REVIEW

File: src/main/java/app/AppBuilder.java
Comment (matt-dahlgren) [inline src/main/java/app/AppBuilder.java:1]: For CodeStyle we should just keep in mind that we should not be useing javax.swing.* star imports do not pass CodeStyle.

**Review by matt-dahlgren — COMMENTED** [review]
Refactoring looking really good, Class names are more comprehensive and help me understand what the inputs are better!

File: src/main/java/app/AppBuilder.java
Comment (atStarling) [reply src/main/java/app/AppBuilder.java:1]: I think we could leave the minor CodeStyle fixes until the last week, so we can get the main code/CA done

**Comment by atStarling** [comment]
I'm a little worried considering the overlap with Pooja's, but I hope that the merge will end up going smoothly. I agree with the refactoring, but it does remind me that we need to deal with the CA mess that is the TeamView class

File: src/main/java/app/AppBuilder.java
Comment (matt-dahlgren) [reply src/main/java/app/AppBuilder.java:1]: That's why I just mentioned to keep in mind. I should have better explained that I meant to just make a comment in the code about doing that later.

**Pull request merged**